secant/Dependencies/ZCashSDKEnvironment.swift

```diff
@@ -19,6 +19,8 @@ fileprivate enum ZcashSDKConstants {
 }
 
 struct ZCashSDKEnvironment {
+    let appBuild: String
```

Comment:
This doesn't seem to belong here, unless we are bundling an SDK version (which wouldn't be a bad idea to be honest)

Reply:
true, I created AppVersionHandler that provides both build + version related the app, we will use this environment to get sdkVersion. Agree it is a valid value to know